### DISCUSSION TITLE
Install Packages

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#787] [Q&A] [answered]
Dear Tim Algoritma,

Mohon bantuannya, saya jalankan beberapa library awalnya bisa. Namun karena ada update packages (otomatis) semua library gagal dijalankan. Saya coba uninstal kemudian diinstal kembali packages yang gagal tadi tetap tidak berhasil.

<img width="673" alt="image" src="https://github.com/teamalgoritma/community/assets/103427989/cad275ce-de24-4636-96ba-3074c17bf019">

Mohon sarannya.

Best regards,
JT

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @joejnt 

Berdasarkan referensi [ini](https://community.rstudio.com/t/tydiverse-error/158729/5), alternatif yang bisa dicoba adalah menginstal `vctrs`

`install.packages('vctrs')`

Semoga jawaban ini bisa membantu.

Best regards, 
Team Algoritma

**Comment by joejnt** [comment]
Dear @saskia-dwi-ulfah 

All done, saya lakukan install.packages("remote") terlebih dahulu, kemudian install_version("vctrs", "0.6.3").

Terima kasih

Best regards,
JT
